core/src/main/res/drawable/ic_danars.xml

```diff
@@ -0,0 +1,64 @@
+<vector android:height="100dp" android:viewportHeight="300"
```

Comment:
should be placed under dana module

Reply:
Hello Milos. :) Thanks for your review.

I'm not sure I get it right. Should the graphic be moved to AndroidAPS / dana / src / main / res /? There is no drawable folder there. Should I create one? Sorry, I'm not a programmer.....